crates/stackable-telemetry/src/instrumentation/axum/mod.rs

```diff
@@ -462,8 +480,8 @@ impl SpanExt for Span {
 
         if new_span_context != current_span_context {
             tracing::trace!(
-                opentelemetry.trace_id.from = ?current_span_context.trace_id(),
-                opentelemetry.trace_id.to = ?new_span_context.trace_id(),
+                { OTEL_TRACE_ID_FROM } = ?current_span_context.trace_id(),
+                { OTEL_TRACE_ID_TO } = ?new_span_context.trace_id(),
```

Comment:
idea (non-blocker): We probably want to link the spans. I can't recall if that is done by Span ID or Trace ID.

See: https://docs.rs/opentelemetry/latest/opentelemetry/trace/trait.Span.html#tymethod.add_link

Reply:
I agree. However, this should be done in a separate PR.